client/components/MiniPledgeForm.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MiniPledgeForm = (props) => (
+  <div className='miniPledgeForm'>
+    <LabelWrapper>Pledge amount</LabelWrapper>
+    <div>
+      <CurrencyWrapper for={`${props.reward.id}pledgeForm`}>$</CurrencyWrapper>
+      <InputWrapper type='text' id={`${props.reward.id}pledgeForm`} defaultValue={`${props.reward.pledgeAmount}`} min={`${props.reward.pledgeAmount}`}></InputWrapper>
+    </div>
```

Comment:
Does eslint complain about not using destructuring when accessing props values on lines 8 and 9?